components/molecule/pagination/src/index.js

```diff
@@ -35,7 +43,11 @@ const MoleculePagination = ({
   nextButtonIcon: NextButtonIcon,
   onSelectPage,
   compressed,
-  hideDisabled
+  hideDisabled,
+  selectedPageButtonDesign,
+  nonSelectedPageButtonDesign,
```

Comment:
doesn't matter much but... why not just `unSelectedPageButtonDesign` or even `pageButtonDesign`?

Reply:
i think pageButtonDesing would infer that it will be the design for all the page buttons and that is  innacurate, thats the point of having two props, one for selected and other one for the non selected pages.

nonSelected and unSelected are synonyms so i don't see a lot of value in the proposed change.

Reply:
IMHO `unSelected` sounds more precise to me